osu.Game/Rulesets/Mods/ModEasy.cs

```diff
@@ -21,7 +22,14 @@ public abstract class ModEasy : Mod, IApplicableToDifficulty, IApplicableFailOve
         public override bool Ranked => true;
         public override Type[] IncompatibleMods => new[] { typeof(ModHardRock) };
 
-        private int retries = 2;
+        [SettingSource("Extra Life count", "Number of extra lives")]
+        public Bindable<int> Retries { get; } = new BindableInt(2)
+        {
+            MinValue = 0,
+            MaxValue = 10
+        };
+
+        private int retries;
```

Comment:
Is this required? / any reason for not using the bindable directly?

Reply:
Decrementing the bindable directly would cause it to set the customized allowed retries count to 0  on the next game you start, thus the existence of this variable which stores the allowed number of retries in the current game

Reply:
Ugh, i'm still not happy with the bindable references being copied across... Alright.